tools/mkexport.sh

```diff
@@ -255,11 +262,12 @@ echo "HOSTINCLUDES     = ${HOSTINCLUDES}" >>"${EXPORTDIR}/scripts/Make.defs"
 echo "HOSTCFLAGS       = ${HOSTCFLAGS}" >>"${EXPORTDIR}/scripts/Make.defs"
 echo "HOSTLDFLAGS      = ${HOSTLDFLAGS}" >>"${EXPORTDIR}/scripts/Make.defs"
 echo "HOSTEXEEXT       = ${HOSTEXEEXT}" >>"${EXPORTDIR}/scripts/Make.defs"
-echo "LDNAME           = ${LDNAME}" >>"${EXPORTDIR}/scripts/Make.defs"
+echo "LDNAME           = ${APPLD}"      >>"${EXPORTDIR}/scripts/Make.defs"
```

Comment:
should we expose both script?

Reply:
Setting LDNAME in `Make.defs` is unnecessary now as makefile system seems working well. However, it doesn't hurt to keep LDNAME in sync with the definitions in `target.cmake`.